flink-python/pyflink/fn_execution/coders.py

```diff
@@ -80,6 +82,64 @@ def __hash__(self):
         return hash(self._flatten_row_coder)
 
 
+class AggregateFunctionInputRowCoder(BaseCoder):
+    """
+    Coder for Aggregate Function Input Row.
+    """
+
+    def __init__(self, flatten_row_coder):
+        self._flatten_row_coder = flatten_row_coder
+
+    def get_impl(self):
+        return coder_impl.AggregateFunctionRowCoderImpl(self._flatten_row_coder.get_impl())
+
+    @staticmethod
+    def from_schema_proto(schema_proto):
+        return AggregateFunctionInputRowCoder(FlattenRowCoder.from_schema_proto(schema_proto))
+
+    def __repr__(self):
+        return 'AggregateFunctionInputRowCoder[%s]' % repr(self._flatten_row_coder)
+
+    def __eq__(self, other):
+        return (self.__class__ == other.__class__
+                and self._flatten_row_coder == other._flatten_row_coder)
+
+    def __ne__(self, other):
+        return not self == other
+
+    def __hash__(self):
+        return hash(self._flatten_row_coder)
+
+
+class AggregateFunctionOutputRowCoder(BaseCoder):
```

Comment:
I think we only introduce a coder named  `AggregateFunctionRowCoder` since the implementations of `AggregateFunctionInputRowCoder` and `AggregateFunctionOutputRowCoder` are the same